tests/test_part_image.py

```diff
@@ -384,7 +384,6 @@ def test_split_mender_conf(
 @pytest.mark.min_mender_version("1.0.0")
 class TestAllPartitionImages:
     @pytest.mark.min_yocto_version("warrior")
-    @pytest.mark.conversion
```

Comment:
The fixture itself should be removed too, no? :thought_balloon:

Reply:
Yes, shouldn't it?

Reply:
No. The fixture needs to remain for other fixtures to use it and modify their logic. It is only the marker and the skip part that we need to get the rid of.